pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1382,6 +1382,10 @@ const (
 	PDLeaderTransferAnnKey = "tidb.pingcap.com/pd-transfer-leader"
 	// TiDBGracefulShutdownAnnKey is the annotation key to graceful shutdown tidb pod by user.
 	TiDBGracefulShutdownAnnKey = "tidb.pingcap.com/tidb-graceful-shutdown"
+	// TiKVEvictLeaderExpirationTimeAnnKey is the annotation key to expire evict leader annotation. Type: time.RFC3339.
+	TiKVEvictLeaderExpirationTimeAnnKey = "tidb.pingcap.com/tikv-evict-leader-expiration-time"
+	// PDLeaderTransferExpirationTimeAnnKey is the annotation key to expire evict leader annotation. Type: time.RFC3339.
+	PDLeaderTransferExpirationTimeAnnKey = "tidb.pingcap.com/pd-evict-leader-expiration-time"
```

Comment:
Do we need to differentiate components using different keys here? Seems the annotation is applied directly to pod rather than the shared `TidbCluster` CR, do we still need to keep track of the component information?

Reply:
Well I would claim that those annotations mentioned above are added by Xiaomou and Hao recently, maybe it's not too late to change the course. I'll leave it to you :)